### PR TITLE
Initial documentation for "asymmetric" dataSource...

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
Here is the initial draft documentation for the "asymmetric" CouchDB dataSource we have discussed previously.

### CODE REVIEW

File: docs/asymmetric.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+The [`kettle.dataSource.CouchDB`](https://github.com/fluid-project/kettle/blob/master/lib/dataSource.js#L399) grade
+included with [Kettle](https://github.com/fluid-project/kettle/) is designed to interact directly with the [CouchDB
+document REST API](https://wiki.apache.org/couchdb/HTTP_Document_API).  It works for cases in which you know the CouchDB
+`_id` of the record up front.  It also stores data wrapped in a `value` element.
+
+This `dataSource` is "asymmetric" because it reads from one type of `dataSource` (the default is
+[`kettle.dataSource.URL`](https://github.com/amb26/kettle/blob/KETTLE-32/lib/dataSource.js#L300)), and writes to another
+(a plain `kettle.dataSource.CouchDB` grade).
+
+The "assymetric" CouchDB dataSource provided by this package is designed for cases in which you wish to read from
```
Comment (amb26) [inline docs/asymmetric.md]: typo asymmetric

**Comment by amb26** [comment]
Thanks, @the-t-in-rtf - this looks like a good plan

File: docs/asymmetric.md (hunk 1 above)
Comment (the-t-in-rtf) [inline docs/asymmetric.md]: Oh, that's unfortunate.  Replaced.

**Comment by the-t-in-rtf** [comment]
Thanks, I'll merge this one and start sketching out the component and tests in the new `gpii-couch-cushion` package once Steve creates it.

**Pull request merged**